shared/extension/FileWatch.hpp

```diff
@@ -76,19 +76,17 @@ namespace NUClear::dsl {
             }
 
             template <typename DSL>
-            static inline ::extension::FileWatch get(threading::Reaction&) {
+            static inline ::extension::FileWatch get(threading::Reaction& /*unused*/) {
 
                 // Get our File Watch store value
-                auto ptr = ::extension::FileWatch::FileWatchStore::value;
+                auto* ptr = ::extension::FileWatch::FileWatchStore::value;
 
                 // If there was something in the store
                 if (ptr) {
```

Comment:
Did clang-tidy not complain about this not being `if (ptr != nullptr) {`?

Reply:
It may have. It makes many many complaints :p 
I'll add this change to the file on #439. Well spotted